spanner/test/opentelemetry/test/ot_traces_test.go

```diff
@@ -33,10 +33,16 @@ import (
 func TestSpannerTracesWithOpenTelemetry(t *testing.T) {
 	ctx := context.Background()
 	te := newOpenTelemetryTestExporter(false, true)
+	trace.OpenTelemetryTracingEnabledMu.RLock()
```

Comment:
I think it would be better for other users to make `OpenTelemetryTracingEnabled` and `OpenTelemetryTracingEnabledMu` private, and move these lock/unlock calls to a setter that controls access to `OpenTelemetryTracingEnabled`. (Be sure to keep the same `Do not access directly` warning.)

Reply:
A setter and a getter, actually.

Reply:
Made the fields `OpenTelemetryTracingEnabled` and `OpenTelemetryTracingEnabledMu` private.
Instead `SetOpenTelemetryTracingEnabledField` can be used to edit this field for unit tests.

Q: Should we also suffix it with UnitTest? something like `SetOpenTelemetryTracingEnabledFieldForUnitTest`. Looks too longer though.